### PR TITLE
en: optimize 2 docs

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/get-started.md [hunk 1]
```diff
@@ -599,37 +563,33 @@ Then access Grafana through <http://${remote-server-IP}:3000>.
 
 For more information about monitoring the TiDB cluster in TiDB Operator, refer to [Deploy Monitoring and Alerts for a TiDB Cluster](monitor-a-tidb-cluster.md).
 
-### Access TiDB Dashboard web UI
-
-You can forward the port for TiDB Dashboard to access TiDB Dashboard web UI locally:
+### Accessing the TiDB Dashboard web UI
```
Comment (ran-huang) [inline en/get-started.md:566]:
```suggestion
### Access the TiDB Dashboard web UI
```

File: en/get-started.md [hunk 2]
```diff
@@ -599,37 +563,33 @@ Then access Grafana through <http://${remote-server-IP}:3000>.
...
 ```bash
 kubectl port-forward --address 0.0.0.0 -n tidb-cluster svc/basic-tidb-dashboard-exposed 12333 > pf12333.out &
 ```
 
 Then access TiDB Dashboard through `http://${remote-server-IP}:12333`.
 
-## Step 5. Upgrade a TiDB cluster
-
-TiDB Operator also makes it easy to perform a rolling upgrade of the TiDB cluster. This section describes how to upgrade your TiDB cluster to the "nightly" release.
+## Step 5: Upgrading a TiDB Cluster
```
Comment (ran-huang) [inline en/get-started.md:584]:
```suggestion
## Step 5: Upgrade a TiDB cluster
```

File: en/get-started.md [hunk 3]
```diff
@@ -599,37 +563,33 @@ Then access Grafana through <http://${remote-server-IP}:3000>.
...
 
-Before that, you need to get familiar with a `kubectl` sub-command `kubectl patch`. It applies a specification change directly to the running cluster resources. There are several different patch strategies, each of which has various capabilities, limitations, and allowed formats. For details, see [Kubernetes Patch](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/)
+The TiDB Operator simplifies the process of performing a rolling upgrade of a TiDB cluster. This section provides instructions on how to upgrade your TiDB cluster to the "nightly" release.
```
Comment (ran-huang) [inline en/get-started.md:586]:
```suggestion
TiDB Operator simplifies the process of performing a rolling upgrade of a TiDB cluster. This section provides instructions on how to upgrade your TiDB cluster to the "nightly" release.
```

File: en/get-started.md [hunk 4]
```diff
@@ -599,37 +563,33 @@ Then access Grafana through <http://${remote-server-IP}:3000>.
...
 
-### Modify the TiDB cluster version
+Before proceeding, it is important to familiarize yourself with the `kubectl patch` sub-command. This command allows you to directly apply changes to the running cluster resources. There are different patch strategies available, each with its own capabilities, limitations, and allowed formats. For more information, refer to the [Kubernetes Patch](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/) documentation.
 
-In this case, you can use a JSON merge patch to update the version of the TiDB cluster to "nightly":
+### Modifying the TiDB Cluster Version
```
Comment (ran-huang) [inline en/get-started.md:590]:
```suggestion
### Modify the TiDB cluster version
```

File: en/get-started.md [hunk 5]
```diff
@@ -644,11 +604,9 @@ tidbcluster.pingcap.com/basic patched
 
 </details>
 
-### Wait for Pods to restart
+### Monitoring Pod Restarts
```
Comment (ran-huang) [inline en/get-started.md:607]:
```suggestion
### Monitor Pod restarts
```

File: en/get-started.md [hunk 6]
```diff
@@ -667,23 +625,21 @@ basic-tikv-0                      1/1     Running       0          4m13s
 
 </details>
 
-### Forward the TiDB service port
-
-After all Pods have been restarted, you can see that the version number of the cluster has changed.
+### Forwarding the TiDB Service Port
```
Comment (ran-huang) [inline en/get-started.md:628]:
```suggestion
### Forward the TiDB service port
```

File: en/get-started.md [hunk 7]
```diff
@@ -667,23 +625,21 @@ basic-tikv-0                      1/1     Running       0          4m13s
...
 
-Note that you need to reset any port forwarding you set up in a previous step, because the pods they forwarded to have been destroyed and recreated.
+Once all Pods have been restarted, you can verify that the cluster's version number has been updated.
 
-{{< copyable "shell-regular" >}}
+Please note that if you had previously set up port forwarding, you will need to reset it as the pods it was forwarding to have been destroyed and recreated.
```
Comment (ran-huang) [inline en/get-started.md:632]:
```suggestion
Note that if you had previously set up port forwarding, you will need to reset it as the Pods it was forwarding to have been destroyed and recreated.
```

File: en/get-started.md [hunk 8]
```diff
@@ -667,23 +625,21 @@ basic-tikv-0                      1/1     Running       0          4m13s
...
 
 ```
 kubectl port-forward -n tidb-cluster svc/basic-tidb 24000:4000 > pf24000.out &
 ```
 
-If the port `24000` is already occupied, you can replace it with an available port.
+If port `24000` is already in use, you can replace it with an available port.
 
-### Check the TiDB cluster version
+### Checking the TiDB Cluster Version
```
Comment (ran-huang) [inline en/get-started.md:640]:
```suggestion
### Check the TiDB cluster version
```

File: en/get-started.md [hunk 9]
```diff
@@ -692,7 +648,7 @@ mysql --comments -h 127.0.0.1 -P 24000 -u root -e 'select tidb_version()\G'
 <details>
 <summary>Expected output</summary>
 
-Note that `nightly` is not a fixed version. Running the command above at a different time might return different results.
+Please note that the version may vary depending on the time the command is run.
```
Comment (ran-huang) [inline en/get-started.md:651]:
```suggestion
Note that the version might vary depending on the time the command is run.
```

File: en/get-started.md [hunk 10]
```diff
@@ -710,107 +666,97 @@ Store: tikv
 
 </details>
 
-## Step 6. Destroy the TiDB cluster and the Kubernetes cluster
-
-After you finish testing, you can destroy the TiDB cluster and the Kubernetes cluster.
+## Step 6: Destroying the TiDB Cluster and the Kubernetes Cluster
```
Comment (ran-huang) [inline en/get-started.md:669]:
```suggestion
## Step 6: Destroy the TiDB cluster and the Kubernetes cluster
```

File: en/get-started.md [hunk 11]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 
-### Destroy the TiDB cluster
+Once you have finished testing, it is time to destroy the TiDB cluster and the Kubernetes cluster.
 
-This section introduces how to destroy a TiDB cluster.
+### Destroying the TiDB Cluster
```
Comment (ran-huang) [inline en/get-started.md:673]:
```suggestion
### Destroy the TiDB cluster
```

File: en/get-started.md [hunk 12]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 
-#### Stop `kubectl` port forwarding
+To destroy the TiDB cluster, follow these steps:
 
-If you still have running `kubectl` processes that are forwarding ports, end them:
+#### Stop `kubectl` Port Forwarding
```
Comment (ran-huang) [inline en/get-started.md:677]:
```suggestion
#### Stop `kubectl` port forwarding
```

File: en/get-started.md [hunk 13]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 
-{{< copyable "shell-regular" >}}
+If you have any running `kubectl` processes that are forwarding ports, make sure to end them by running the following command:
 
 ```shell
 pgrep -lfa kubectl
 ```
 
-#### Delete the TiDB cluster
+#### Delete the TiDB Cluster
```
Comment (ran-huang) [inline en/get-started.md:685]:
```suggestion
#### Delete the TiDB cluster
```

File: en/get-started.md [hunk 14]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 
 ```shell
 kubectl delete tc basic -n tidb-cluster
 ```
 
-The `tc` in this command is a short name for tidbclusters.
+In this command, `tc` is a shorthand for `tidbclusters`.
 
-#### Delete TiDB monitoring services
+#### Delete TiDB Monitoring Services
```
Comment (ran-huang) [inline en/get-started.md:695]:
```suggestion
#### Delete TiDB monitoring services
```

File: en/get-started.md [hunk 15]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 
-{{< copyable "shell-regular" >}}
+To delete the TiDB monitoring services, run the following command:
 
 ```shell
 kubectl delete tidbmonitor basic -n tidb-cluster
 ```
 
-#### Delete PV data
+#### Delete PV Data
```
Comment (ran-huang) [inline en/get-started.md:703]:
```suggestion
#### Delete PV data
```

File: en/get-started.md [hunk 16]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 
-If your deployment has persistent data storage, deleting the TiDB cluster does not remove the data in the cluster. If you do not need the data, run the following commands to clean it:
-
-{{< copyable "shell-regular" >}}
+If your deployment includes persistent data storage, deleting the TiDB cluster will not remove the data in the cluster. If you do not need the data, you can clean it by running the following commands:
```
Comment (ran-huang) [inline en/get-started.md:705]:
```suggestion
If your deployment includes persistent data storage, deleting the TiDB cluster does not remove the data in the cluster. If you do not need the data, you can clean it by running the following commands:
```

File: en/get-started.md [hunk 17]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 
 ```shell
 kubectl delete pvc -n tidb-cluster -l app.kubernetes.io/instance=basic,app.kubernetes.io/managed-by=tidb-operator && \
 kubectl get pv -l app.kubernetes.io/namespace=tidb-cluster,app.kubernetes.io/managed-by=tidb-operator,app.kubernetes.io/instance=basic -o name | xargs -I {} kubectl patch {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
 ```
 
-#### Delete namespaces
-
-To ensure that there are no lingering resources, delete the namespace used for your TiDB cluster.
+#### Delete Namespaces
```
Comment (ran-huang) [inline en/get-started.md:712]:
```suggestion
#### Delete namespaces
```

File: en/get-started.md [hunk 18]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 
-{{< copyable "shell-regular" >}}
+To ensure that there are no remaining resources, delete the namespace used for your TiDB cluster by running the following command:
 
 ```shell
 kubectl delete namespace tidb-cluster
 ```
 
-### Destroy the Kubernetes cluster
+### Destroying the Kubernetes Cluster
```
Comment (ran-huang) [inline en/get-started.md:720]:
```suggestion
### Destroy the Kubernetes cluster
```

File: en/get-started.md [hunk 19]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
-``` shell
+```shell
 minikube delete
 ```
 
 </div>
 </SimpleTab>
 
-## See also
+## See Also
```
Comment (ran-huang) [inline en/get-started.md:746]:
```suggestion
## See also
```

File: en/get-started.md [hunk 20]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 On public clouds:
 
-- [Deploy TiDB on AWS EKS](deploy-on-aws-eks.md)
-- [Deploy TiDB on Google Cloud GKE](deploy-on-gcp-gke.md)
-- [Deploy TiDB on Azure AKS](deploy-on-azure-aks.md)
-- [Deploy TiDB on Alibaba Cloud ACK](deploy-on-alibaba-cloud.md)
+- [Deploying TiDB on AWS EKS](deploy-on-aws-eks.md)
+- [Deploying TiDB on Google Cloud GKE](deploy-on-gcp-gke.md)
+- [Deploying TiDB on Azure AKS](deploy-on-azure-aks.md)
+- [Deploying TiDB on Alibaba Cloud ACK](deploy-on-alibaba-cloud.md)
```
Comment (ran-huang) [inline en/get-started.md:755]:
```suggestion
- [Deploy TiDB on AWS EKS](deploy-on-aws-eks.md)
- [Deploy TiDB on Google Cloud GKE](deploy-on-gcp-gke.md)
- [Deploy TiDB on Azure AKS](deploy-on-azure-aks.md)
- [Deploy TiDB on Alibaba Cloud ACK](deploy-on-alibaba-cloud.md)
```

File: en/get-started.md [hunk 21]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 
 In a self-managed Kubernetes cluster:
 
-- Familiarize yourself with [Prerequisites for TiDB on Kubernetes](prerequisites.md)
-- [Configure the local PV](configure-storage-class.md#local-pv-configuration) for your Kubernetes cluster to achieve high performance for TiKV
+- Familiarize yourself with the [Prerequisites for TiDB on Kubernetes](prerequisites.md)
+- [Configure the Local PV](configure-storage-class.md#local-pv-configuration) for your Kubernetes cluster to achieve high performance for TiKV
```
Comment (ran-huang) [inline en/get-started.md:760]:
```suggestion
- [Configure the local PV](configure-storage-class.md#local-pv-configuration) for your Kubernetes cluster to achieve high performance for TiKV
```

File: en/get-started.md [hunk 22]
```diff
@@ -710,107 +666,97 @@ Store: tikv
...
 - [Deploy TiDB Operator on Kubernetes](deploy-tidb-operator.md)
-- [Deploy TiDB on General Kubernetes](deploy-on-general-kubernetes.md)
\ No newline at end of file
+- [Deploy TiDB on a General Kubernetes Cluster](deploy-on-general-kubernetes.md)
```
Comment (ran-huang) [inline en/get-started.md:762]:
```suggestion
- [Deploy TiDB on General Kubernetes](deploy-on-general-kubernetes.md)
```

File: en/configure-storage-class.md [hunk 23]
```diff
@@ -133,9 +125,9 @@ The `/mnt/ssd`, `/mnt/sharedssd`, `/mnt/monitoring`, and `/mnt/backup` directori
               mountDir: /mnt/backup
         ```
 
-        For more configuration about local-volume-provisioner, refer to [Configuration](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/provisioner.md#configuration).
+        For more configuration options for the local-volume-provisioner, refer to the [Configuration](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/provisioner.md#configuration) documentation.
```
Comment (lilin90) [inline en/configure-storage-class.md:128]: Let's use "document" when we refer to one file or page.
```suggestion
        For more configuration options for the local-volume-provisioner, refer to the [Configuration](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/provisioner.md#configuration) document.
```

File: en/configure-storage-class.md [hunk 24]
```diff
@@ -168,40 +160,34 @@ The `/mnt/ssd`, `/mnt/sharedssd`, `/mnt/monitoring`, and `/mnt/backup` directori
...
-    `local-volume-provisioner` creates a PV for each mounting point under the discovery directory.
+    The `local-volume-provisioner` creates a PV for each mounting point under the discovery directory.
 
     > **Note:**
     >
-    > If no mount point is in the discovery directory, no PV is created and the output is empty.
+    > If there are no mount points in the discovery directory, no PV is created and the output is empty.
 
-For more information, refer to [Kubernetes local storage](https://kubernetes.io/docs/concepts/storage/volumes/#local) and [local-static-provisioner document](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#overview).
+For more information, refer to the [Kubernetes local storage](https://kubernetes.io/docs/concepts/storage/volumes/#local) and [local-static-provisioner document](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#overview).
```
Comment (lilin90) [inline en/configure-storage-class.md:182]:
```suggestion
For more information, refer to the [Kubernetes local storage](https://kubernetes.io/docs/concepts/storage/volumes/#local) and [local-static-provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#overview) documents.
```

File: en/configure-storage-class.md [hunk 25]
```diff
@@ -210,29 +196,25 @@ Steps of offline deployment is same as online deployment, except the following:
...
     Copy the `local-volume-provisioner-v2.3.4.tar` file to the server, and execute the `docker load` command to load the file on the server:
 
-    {{< copyable "shell-regular" >}}
-
     ```shell
     docker load -i local-volume-provisioner-v2.3.4.tar
     ```
 
-### Best practices
+### Best Practices
```
Comment (lilin90) [inline en/configure-storage-class.md:203]:
```suggestion
### Best practices
```

File: en/configure-storage-class.md [hunk 26]
```diff
@@ -210,29 +196,25 @@ Steps of offline deployment is same as online deployment, except the following:
...
 
-- A local PV's path is its unique identifier. To avoid conflicts, it is recommended to use the UUID of the device to generate a unique path.
-- For I/O isolation, a dedicated physical disk per PV is recommended to ensure hardware-based isolation.
-- For capacity isolation, a partition per PV or a physical disk per PV is recommended.
+- The unique identifier for a local PV is its path. To avoid conflicts, it is recommended to generate a unique path using the UUID of the device.
+- To ensure I/O isolation, it is recommended to use a dedicated physical disk per PV for hardware-based isolation.
+- For capacity isolation, it is recommended to use either a partition per PV or a physical disk per PV.
 
-For more information on local PV on Kubernetes, refer to [Best Practices](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/best-practices.md).
+For more information on local PV on Kubernetes, refer to the [Best Practices](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/best-practices.md) documentation.
```
Comment (lilin90) [inline en/configure-storage-class.md:209]:
```suggestion
For more information on local PV on Kubernetes, refer to the [Best Practices](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/best-practices.md) document.
```

File: en/configure-storage-class.md [hunk 27]
```diff
@@ -260,22 +240,18 @@ In general, after a PVC is no longer used and deleted, the PV bound to it is rec
...
-
-    {{< copyable "shell-regular" >}}
+2. Set the reclaim policy of the PV to `Delete`. This automatically deletes and reclaims the PV.
 
     ```shell
     kubectl patch pv ${pv_name} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
     ```
 
-For more details, refer to [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/).
+For more details, refer to the [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) documentation.
```
Comment (lilin90) [inline en/configure-storage-class.md:257]:
```suggestion
For more details, refer to the [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) document.
```

**Pull request merged**